runtime/metricsview/astexpr.go

```diff
@@ -210,7 +224,12 @@ func (b *sqlExprBuilder) writeBinaryCondition(exprs []*Expression, op Operator)
 
 		// If not unnested, write the expression as-is
 		if !unnest {
```

Comment:
1. The comment here is no longer correct – because it is not returned "as-is" for lookups
2. What if a lookup is used together with `unnest`? Is it possible? If not, can it return an error if it happens?